client/src/app/app.component.html

```diff
@@ -1,9 +1,40 @@
 <mat-sidenav-container class="sidenav-container">
-<mat-toolbar color="primary">
-  <span class="app-title">{{title}}</span>
-</mat-toolbar>
+  <mat-toolbar fxLayout="row" color="primary">
+    <!--The container takes up 10% of the row-->
+    <div class="menu-button-container">
+      <button class="menu-button" mat-button [matMenuTriggerFor]="appMenu">
+        <mat-icon>
+          reorder
```

Comment:
The `menu` icon would be more appropriate here.

```suggestion
          menu
```

Reply:
Wilco!